### ISSUE TITLE
Segmentation fault (core dumped) when training and CommonTest Failed

### ISSUE THREAD

**Issue by ZhilunX** [BVLC/caffe#312]
Hi. I just train my model. It seems everything goes well. But when testing the net. it outputs Segmentation fault. I don't know what's the problem. Thanks.

I0410 19:44:32.296176 26997 solver.cpp:36] Solver scaffolding done.
I0410 19:44:32.296200 26997 solver.cpp:47] Solving CaffeNet
I0410 19:49:34.169067 26997 solver.cpp:208] Iteration 500, lr = 0.01
I0410 19:49:34.172401 26997 solver.cpp:65] Iteration 500, loss = 0
I0410 19:54:43.192100 26997 solver.cpp:208] Iteration 1000, lr = 0.01
I0410 19:54:43.195472 26997 solver.cpp:65] Iteration 1000, loss = 0
I0410 19:54:43.195485 26997 solver.cpp:87] Iteration 1000, Testing net
Segmentation fault (core dumped)
